tests/test_views.py

```diff
@@ -557,6 +561,119 @@ def test_detail_view_staff_all_users(self, mock_update):
         assert mock_update.call_count == 0
         assert models.StaleCompletion.objects.filter(resolved=False).count() == 2
 
+    def _create_cohort(self, owner, users):
+        """
+        Create and populate a user group, as well as a cohort.
+        """
+        user_group = empty_compat.course_user_group().objects.create(
+            name='test',
+            course_id=self.course_key,
+            group_type='cohort',
+        )
+        user_group.users.add(*users)
+        owner.cohortmembership_set.add(
+            empty_compat.cohort_membership_model().objects.create(
+                course_user_group=user_group,
+                user=owner,
+                course_id=self.course_key,
+            ),
+        )
+
+    @XBlock.register_temp_plugin(StubCourse, 'course')
+    @XBlock.register_temp_plugin(StubSequential, 'sequential')
+    @XBlock.register_temp_plugin(StubHTML, 'html')
+    def test_stat_view_course_no_cohorts(self):
+        response = self.client.get(self.get_course_stat_url(
+            'edX/toy/2012_Fall',
+            cohorts=1,
+            exclude_roles='staff',
+        ))
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
Ah.  I see it's doing the right thing with no cohort provided.  Then the above doc fix should just be in the docs. :)